### ISSUE TITLE
couchpotato and sonar web links broken

### ISSUE THREAD

**Issue by phillxnet** [rockstor/rockon-registry#20]
When clicking on the couchpotato 'name' link in the available Rock-ons page there is no external url but a relative one to the installed system, ie on a system named rockcube.lan we get:
```
https://rockcube.lan/<https://hub.docker.com/r/linuxserver/couchpotato></a>
```
The same issue exists with the sonar rock-on:
```
https://rockcube.lan/<https://hub.docker.com/r/linuxserver/sonarr></a>
```

**Comment by tronder88** [comment]
Think issue can be closed now. I have informed Suman about the issue and he has corrected them.

**Comment by phillxnet** [comment]
@tronder88 Thanks for the heads up.
Closing as fixed by #22 and https://github.com/rockstor/rockon-registry/commit/09f36ee530a1005129cbb7b02412ddde7e85d9a6
